src/ScrollToTop.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router';
+
+class ScrollToTop extends Component {
+  componentDidUpdate(prevProps) {
+    if (this.props.location !== prevProps.location) {
+      window.scrollTo(0, 0);
+    }
+  }
+
+  render() {
+    return this.props.children;
+  }
+}
+
+ScrollToTop.propTypes = {
+  children: React.PropTypes.func.isRequired,
```

Comment:
So, you can ditch `React.` in front of both instances of `PropTypes.`

Additionally, children and location are both actually objects!

Lastly, to get past the linter, you'll want to add `// eslint-disable-line` to the end of each PropType definition.